src/sbt-test/sbt-scalafix/cross-build/scala213/src/main/scala/Source3.scala

```diff
@@ -0,0 +1,7 @@
+open class Source3
+
+object Source3 {
+  implicit class XtensionVal(val str: String) extends AnyVal {
```

Comment:
just amended the last commit yo move this inside an object to avoid
```
[info] [error] /tmp/sbt_5a68d1e4/cross-build/scala213/src/main/scala/Source3.scala:3:16: XtensionVal is already defined as (compiler-generated) method XtensionVal
```